recipes/jupyterlite-javascript-kernel/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "jupyterlite-javascript-kernel" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlite_javascript_kernel-{{ version }}.tar.gz
+  sha256: 297116be6642c2f72b0b0bf3d6505efcd792b322139174d0107ae7ec4daa64f4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling >=1.6.3,<2
+    - jupyterlab >=3.5.3,<3.6.0
+    - pip
+  run:
+    - python >=3.7
```

Comment:
depends on `jupyterlite-core`

Reply:
I guess, although like for regular JupyterLab they don't depend on JupyterLab.

Reply:
and this one does not depend on `jupyterlite_core` on the Python side (no addon).

Reply:
but it almost certainly _isn't_ going to work in `jupyterlite 0.2.0`

Reply:
Then it would be fine to add the dependency just as a safety net for now :+1:

Reply:
cool, put it back as plain-old `run` (instead of `run_constrained`), and removed the TODO stuff: [b751b97](https://github.com/conda-forge/staged-recipes/pull/22475/commits/b751b97ffce49dec5d955264cd634b5ee00c8d1c)

once that's green will ask for review from the `staged-recipes` folk :sweat_smile: